.eslintrc.js

```diff
@@ -6,10 +6,12 @@ module.exports = {
   },
   plugins: ['@typescript-eslint/eslint-plugin'],
   extends: [
-    'plugin:@typescript-eslint/eslint-recommended',
```

Comment:
According to [the documentation](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin/src/configs), this line is not necessary as it's automatically included with `plugin:@typescript-eslint/recommended` which is included below.

Here's a screenshot for your convenience.

![image](https://user-images.githubusercontent.com/613647/91586405-9a64a300-e955-11ea-938f-9ed52072c464.png)